internal/servers/controller/handlers/authmethods/password.go

```diff
@@ -140,6 +166,34 @@ func (s Service) authenticateWithPwRepo(ctx context.Context, scopeId, authMethod
 	return prot, nil
 }
 
+func validateAuthenticatePasswordRequest(req *pbs.AuthenticateRequest) error {
```

Comment:
perhaps check for nil req?

Reply:
I don't think grpc can ever give us a nil proto. Regardless, it's validated by the first non-password-specific call.